resources/idlharness.js

```diff
@@ -218,6 +226,81 @@ IdlArray.prototype.add_untested_idls = function(raw_idls, options)
     this.internal_add_idls(parsed_idls, options);
 };
 
+//@}
+IdlArray.prototype.add_dependency_idls = function(raw_idls, options)
+//@{
+{
+    const parsed_idls = WebIDL2.parse(raw_idls);
+    const new_options = { only: [] }
+    
+    const all_deps = new Set();
+    Object.values(this["inheritance"]).forEach(v => all_deps.add(v));
+    Object.values(this["implements"]).forEach(v => all_deps.add(v));
```

Comment:
Should "includes" also be processed here? That's for mixins, added in https://github.com/w3c/web-platform-tests/pull/8727

Reply:
I treat it as a dep type below, so yeah, I think it should.